projects/v3/src/app/pages/chat/chat-room/chat-room.component.ts

```diff
@@ -353,6 +355,8 @@ export class ChatRoomComponent implements OnInit {
               channelUuid: this.chatChannel.uuid,
               readcount: messageIds.length
             });
+            // after messages read need to update chat notification data on notification service
+            this.notificationsService.getChatMessage().subscribe();
```

Comment:
hi @sasangachathumal this one maybe can move to 'chat-badge-update' listener, or add another listener at a page where the notifications are gathered. what do you think?

because the line 359 here, look just like another broadcaster, which it means we have 2 broadcaster "chat-badge-update" and "getChatMessage" to force notification update. let's discuss tomorrow.